edgexfoundry/device-sdk-go
# build: Consume latest go-mod-bootstrap with RemoveHosts flag

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
run edgex non-secure stack
build device simple
run ./device-simple -cp -r -rsh=172.26.113.174,172.26.113.174,0.0.0.0`
verify the service bootstraps successfully
verify the log contains the following:
- Setting config Provider host to 172.26.113.174
- Using Configuration provider (consul) from: http://172.26.113.174:8500
- Using Registry (consul) from http://172.26.113.174:8500
- Connected to redis Message Bus @ redis://172.26.113.174:6379
-  Using REST for 'core-metadata' clients @ http://172.26.113.174:59881
- Using REST for 'core-data' clients @ http://172.26.113.174:59880
- Using REST for 'support-notifications' clients @ http://172.26.113.174:59860
- Starting HTTP Web Server on address 0.0.0.0:59740

run `REMOTE_SERVICE_HOSTS=172.26.113.174,172.26.113.174,0.0.0.0 ./app-new-service -cp -r`
verify the service bootstraps successfully
verify the log contains the same above contents:

## New Dependency Instructions (If applicable)
